libsolidity/parsing/Token.cpp

```diff
@@ -80,8 +112,35 @@ char const Token::m_tokenType[] =
 {
 	TOKEN_LIST(KT, KK)
 };
-Token::Value Token::fromIdentifierOrKeyword(const std::string& _name)
+tuple<Token::Value, string> Token::fromIdentifierOrKeyword(const string& _literal)
 {
+	string token = _literal;
+	string details;
+	if (_literal == "uintM" || _literal == "intM" || _literal == "fixedMxN" || _literal == "ufixedMxN" || _literal == "bytesM")
```

Comment:
This suggestion throws errors...along with trying it with just Identifier subbed for the left most argument due to a redefinition error.